.github/workflows/f5-cla.yml

```
@@ -2,9 +2,9 @@
 name: F5 CLA
 on:
   issue_comment:
-    types: [created]
+    types: [ created ]
   pull_request_target:
-    types: [opened, closed, synchronize]
+    types: [ opened, closed, synchronize ]
```

```suggestion
    types: [opened, closed, synchronize]
```
Minor style nit (for consistency across the master file too). No empty space between list brackets.